assets/css/main.scss

```diff
@@ -12,4 +12,12 @@
 }
 .left-space{
 	padding-right:4em;
+}
+.collection-list{
+	a:link,:visited	{
+		color: #0092ca;
+	}
+	a:hover	{
+		color: #03A9F4;
+	}
 }
```

Comment:
By default, links have a dull color. By adding css, I am changing link color to blue. On hover, the color of link changes and gives a glowing effect to the link. You can check default link color on gsoc19 page.

Reply:
I totally agree with you. However, I would say you should create a new issue to solve this, and not mix it with this one. Hope that makes sense.

Reply:
Yes it makes sense. I wrote this code specifically for collections. We can still create a new issue because this css needs to be applied to all links.

Reply:
Great, so it should be removed from this PR.

Reply:
Ok. I will work on it.

Reply:
I have removed CSS from this PR.